store/tikv/logutil/tracing.go

```diff
@@ -0,0 +1,46 @@
+// Copyright 2021 PingCAP, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package logutil
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/opentracing/opentracing-go"
+	"github.com/opentracing/opentracing-go/log"
+)
+
+// TraceEventKey presents the TraceEventKey in span log.
+var TraceEventKey = "event"
```

Comment:
Could we remove the same logic in `tidb/util/logutil`, and import `tikv/logutil` instead if needed?

Reply:
It doesn't feel very appropriate. Considering that `tidb` and `client-go` will be separated in the future, it is better that the log libraries do not depend on each other.
Now the code copy is made for compatibility reason. I envision that in the future we can make some adjustments to the client's log library (for example, no longer use`pingcap/log`).